### ISSUE TITLE
Missing accessibility identifiers for settings elements

### ISSUE THREAD

**Issue by dragosb01** [mozilla-mobile/firefox-ios#19729]
There are some elements on settings that are missing accessibility identifiers.
This is a blocker for automation to fully validate these elements.
- offer to open copied links
- show link previews
- close private tabs
- Privacy Policy
- Send Feedback
- Help
- Rate on App Store
- Licenses
- Your Rights
<img width="828" alt="Screenshot 2024-04-10 at 14 04 27" src="https://github.com/mozilla-mobile/firefox-ios/assets/134391433/96e36893-b161-46c2-870b-fa47df5bd774">
<img width="911" alt="Screenshot 2024-04-10 at 14 03 43" src="https://github.com/mozilla-mobile/firefox-ios/assets/134391433/c6d74ff0-48ba-4c0d-a1d8-cad3c654d767">
<img width="379" alt="Screenshot 2024-04-10 at 14 03 27" src="https://github.com/mozilla-mobile/firefox-ios/assets/134391433/6006e681-4991-4365-ab66-a18eafe07f0e">
<img width="854" alt="Screenshot 2024-04-10 at 14 02 08" src="https://github.com/mozilla-mobile/firefox-ios/assets/134391433/9e576b2c-c6ff-4099-b224-0da2b57ee6e7">
<img width="375" alt="Screenshot 2024-04-10 at 14 01 57" src="https://github.com/mozilla-mobile/firefox-ios/assets/134391433/b1ef567a-4bdb-4dc4-8ae7-c2a4f5781d2b">

**Comment by data-sync-user** [comment]
➤ Dragos Bigu commented:

Norberto Andres Furlan requesting a priority for this task.

**Comment by data-sync-user** [comment]
➤ Alexandru Farcasanu commented:

Should be verified by Dragos Bigu

**Comment by data-sync-user** [comment]
➤ Dragos Bigu commented:

Identifiers added, closing ticket.
